taxonomy/apis.py

```diff
@@ -0,0 +1,226 @@
+"""APIs for lore taxonomy application"""
+
+from taxonomy.models import Vocabulary, Term
+from learningobjects.models import LearningObject, LearningObjectType
+
+
+def get_vocabulary(vocabulary_id):
+    """
+    Lookup vocabulary given its id
+    :param vocabulary_id: Vocabulary id
```

Comment:
Totally open for discussion, but we've actually been using https://pypi.python.org/pypi/sphinxcontrib-napoleon/0.3.3 and the Google style guide for doc strings: http://google-styleguide.googlecode.com/svn/trunk/pyguide.html

I just tend to prefer the look, but we can change our minds, no problem.  Also the params are missing types, like `str`, `int`, etc.